config/config.go

```diff
@@ -412,6 +413,55 @@ type GlobalConfig struct {
 	// Keep no more than this many dropped targets per job.
 	// 0 means no limit.
 	KeepDroppedTargets uint `yaml:"keep_dropped_targets,omitempty"`
+	// The protocols to negotiate during a scrape. It tells clients what
+	// protocol are accepted by Prometheus and with what preference (most wanted is first).
+	// Supported values (case insensitive): PrometheusProto, OpenMetricsText, PrometheusText.
+	AcceptScrapeProtocols []ScrapeProtocol `yaml:"accept_scrape_protocols,omitempty"`
```

Comment:
After some time, I think I will simplify this field to "scrape_protocols" 🤔 wdyt? No need for "accept" word - it might only confuse.
```suggestion
	ScrapeProtocols []ScrapeProtocol `yaml:"scrape_protocols,omitempty"`
```